.github/CODEOWNERS

```diff
@@ -7,5 +7,5 @@
 # These owners will be the default owners for everything in the repo.
 # Unless a later match takes precedence, they will be requested for
 # review when someone opens a pull request.
-@Alan-Jowett @dthaler @dv-msft @LakshK98 @matthewige @mikeagun @mtfriesen
-@poornagmsft @saxena-anurag @shankarseal @vpidatala94
\ No newline at end of file
+* @Alan-Jowett @dthaler @dv-msft @LakshK98 @matthewige @mikeagun @mtfriesen
+@poornagmsft @saxena-anurag @shankarseal @vpidatala94
```

Comment:
Why no * for the second line? 

The second line matches on a files called "@poornagmsft"

Reply:
I believe they need to all be on the same line.